target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -70,6 +70,9 @@ tplink,re650-v1)
 xzwifi,creativebox-v1)
 	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "wan"
 	;;
+xiaomi,redmi-router-ac2100)
+	ucidef_set_led_netdev "wan" "wan" "$boardname:white:wan" "wan"
```

Comment:
led name should be `rm2100:white:wan` according to dts

Reply:
Thanks for spotting.
I'd rather prefer the DTS names to match the device name, though.